packages/gatsby-plugin-gatsby-cloud/package.json

```diff
@@ -51,11 +52,11 @@
   },
   "sideEffects": false,
   "scripts": {
-    "build": "babel src --out-dir . --ignore \"**/__tests__\" && npm run clean && npm run copy-type-declarations",
+    "build": "babel src --out-dir . --ignore \"**/__tests__\" --extensions \".tsx,.ts,.js\" --source-maps && npm run clean && npm run copy-type-declarations",
```

Comment:
I think we can leave this out of this PR since TS conversion will come in another PR 👍